src/constants/settings.js

```diff
@@ -36,4 +37,28 @@ module.exports = {
 
   // elements settings properties,
   VALID_MODES: ["folder", "file", "full"],
+
+  VALID_DEPENDENCY_NODE_KINDS: ["value", "type"],
+  PREDEFINED_DEPENDENCY_NODES: {
+    import: [
+      // import x from 'source'
+      { selector: "ImportDeclaration:not([importKind=type]) > Literal", kind: "value" },
+      // import type x from 'source'
+      { selector: "ImportDeclaration[importKind=type] > Literal", kind: "type" },
+    ],
```

Comment:
__suggestion__: Would it be better to move this dependency node to a different constant, so users are not able to select it again as an additional dependency node, producing that it could be processed twice? What about `DEFAULT_DEPENDENCY_NODES`, for example?